### DISCUSSION TITLE
Wind speed m/s

### DISCUSSION THREAD

**Discussion by mikkoma** [rocksdanister/weather#58] [Ideas]
First of all I really like the design and features but would love to see option to show wind speed meters per second while still having temperature as celsius and other as metric too.

**Comment by mikkoma** [comment]
Also option to autorfresh when in front and in background would be nice. 
Does it now refresh its data automatically or do I have to click that refresh timeinfo?

**Reply by rocksdanister** [reply]
It refreshes when opening the app if the cache file is an hour old.

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/weather/releases/tag/v1.0.33.0
